README.md

```diff
@@ -3,6 +3,8 @@ KubeVirt Web User Interface
 
 [![Build Status](https://travis-ci.org/kubevirt/web-ui.svg?branch=master)](https://travis-ci.org/kubevirt/web-ui)
 
+### This project is deprecated in favour of [OpenShift Console](https://github.com/openshift/console) where it is included as plugin
```

Comment:
`favour` ==> `favor`
Dot at the end of the sentence.

Reply:
fixed